### ISSUE TITLE
Babel Optimization Error: Module Size Exceeding 500KB

### ISSUE THREAD

**Issue by 0-don** [x64Bits/solid-icons#61]
### Description

When building a SolidJS project using `solid-icons`, Babel throws an optimization error indicating that the styling of some modules exceeds the maximum size of 500KB. This is occurring specifically with `solid-icons/fa/index.js` and `solid-icons/tb/index.js`. The error message is repeated multiple times, suggesting that the issue persists through multiple attempts to compile.

### Steps to Reproduce

1. Set up a SolidJS project with `@solidjs/start`.
2. Include the `solid-icons` package in the dependencies.
3. Run the build script.

### Expected Behavior

The build should complete without errors, and Babel should handle the module size appropriately, possibly by splitting or tree-shaking.

### Actual Behavior

The following error is received:
```bash
[BABEL] Note: The code generator has deoptimised the styling of /home/don/MEGA/Projects/coding.global-web/node_modules/solid-icons/tb/index.js as it exceeds the max of 500KB.
[BABEL] Note: The code generator has deoptimised the styling of /home/don/MEGA/Projects/coding.global-web/node_modules/solid-icons/fa/index.js as it exceeds the max of 500KB.
```

### Environment

- SolidJS Version: 1.8.15
- Solid Icons Version: 1.1.0
- Node Version: 18
- Package Manager: [npm/yarn]
- Operating System: [e.g., macOS/Windows/Linux]

### Possible Solution

Could this be an issue with how the icons are being imported or bundled? Is there a recommended way to import `solid-icons` that could avoid this problem?

### Additional Context

![image](https://github.com/x64Bits/solid-icons/assets/70039285/25585852-aa4f-4e4f-ad15-ee7374c31539)

**Comment by x64Bits** [comment]
Hi @0-don thanks for sharing, solid-start receive new updates recently, i will check this and go back with update soon.

**Comment by sparklog** [comment]
In dev mode, Babel shows the error as above. However, when I built it, app works well. I think 'tree shake' function solved this issue during the build process. Am I right?

**Comment by diegovfeder** [comment]
Just noticed the same issue here.

**Comment by sysmat** [comment]
same hare

**Comment by jcalfee** [comment]
The one fa index file is over 2k lines and they all look like this: 
```js
export declare const FaSolidY: IconTypes;
export declare const FaSolidYenSign: IconTypes;
export declare const FaSolidYinYang: IconTypes;
export declare const FaSolidZ: IconTypes;
```

There is no way to import one of these.  Perhaps all these icons need separate files, but that is a pretty rough way to fix the issue.  I think the code generator is going to need to be a bit smarter.

**Comment by x64Bits** [comment]
I agree, in fact I've been iterating recently on how to make icon generation more optimized, going for files separated by icons and their types sounds the most reasonable, I remember that I decided to export the types of each icon name in that way because that way VSCode suggested the names automatically.

It's possible that the reason for doing it that way is that it's not logical for the library's operation.
